assets/scss/sections/_baker-croissant.scss

```diff
@@ -0,0 +1,49 @@
+.bakery-croissant{
+    display:flex;
+    padding:100px 0;
+    box-sizing: border-box;
+    height:100vh;
+    .bakery-croissant-boxes-wrapper{
+        display:flex;
+        justify-content: center;
+        align-items: center;
+        height:100%;
+        .bakery-croissant-box{
+            width:50%;
+            height:100%;
+            display:flex;
+            flex-flow: column;
+            justify-content:center;
+            overflow: hidden;
+            position: relative;
+            box-sizing: border-box;
+            img{
+                height:auto;
+                width:100%;
+                position: absolute;
+                top:50%;
+                transform:translateY(-60%)
```

Comment:
Jeśli dałeś top:50% i -60% translateY to coś na pewno jest nie tak.
Poza tym do wywalenia.

Reply:
Poprawione